tests/flow/test_with_clause.py

```diff
@@ -267,3 +267,24 @@ def test12_cartesian_product_reset_single_response(self):
         query = """WITH 1 AS x MATCH (a:label_a), (b:label_b) RETURN a.v, b.v"""
         actual_result = redis_graph.query(query)
         self.env.assertEqual(len(actual_result.result_set), 36)
+
+    def test_13_undefined_variables(self):
```

Comment:
Please migrate to `test_query_validation.py`
See if you can extend to queries using only the RETURN clause, e.g. `RETURN a AS a`